GiniCapture/Classes/Core/Screens/Camera/CameraViewController.swift

```diff
@@ -446,6 +446,7 @@ extension CameraViewController: CameraPreviewViewControllerDelegate {
         cameraButtonsViewController.isFlashSupported = camera.isFlashSupported
         cameraButtonsViewController.view.setNeedsLayout()
         cameraButtonsViewController.view.layoutIfNeeded()
+        viewController.stopLoadingIndicator()
```

Comment:
Stopping the loading indicator could be done by the `CameraPreviewViewController` itself before or after calling `self.delegate?.cameraDidSetUp(self, camera: self.camera)` on line 169. This would encapsulate the loading indicator starting/stopping logic in `CameraPreviewViewController`.